Added VERSION SQL function

samie opened the pull request:
Coming from MySQL I was missing database metadata functions. This is one of the simplest ones and might be handy. 
I'm not really familiar with JavaCC, but followed the implementation of similar RANDOM SQL function to add VERSION to vocabulary. The version information itself is added from pom.xml during build to the messages using resource filtering.

simoc commented:
Please also add the new VERSION function to the list of SQL functions in markdown file `docs/doc.md`.

Something like:

VERSION() Returns string containing Csvjdbc version number

samie commented:
I added that as documentation. Also noticed that the Driver and DatabaseMetadata should return the real version instead of hard-coded. I moved the parsing logic to CsvResources to make it usable everywhere.
